### ISSUE TITLE
Type of expression is ambiguous without more context

### ISSUE THREAD

**Issue by jobinsjohn** [wxxsw/GSMessages#30]
Today after the updation i am getting an issue like this

<img width="1409" alt="screen shot 2017-12-12 at 1 52 00 pm" src="https://user-images.githubusercontent.com/3398498/33878067-d0fa8354-df43-11e7-8a4e-810f0dea42d1.png">

**Comment by wxxsw** [comment]
I want to see the full options parameter, thanks

**Comment by jobinsjohn** [comment]
self.showMessage("MESSAGE TEXT", type: .warning, options: [
                                .animation(.slide),
                                .animationDuration(0.3),
                                .autoHide(true),
                                .autoHideDelay(3.0),
                                .height(60.0),
                                .hideOnTap(true),
                                .position(.top),
                                .textAlignment(.center),
                                .textColor(UIColor(red: 239.0/255, green: 169.0/255, blue: 4.0/255, alpha: 0.95)),
                                .textNumberOfLines(1),
                                .padding(.init(top: 10, left: 30, bottom: 10, right: 30))
                                ])

**Comment by wxxsw** [comment]
The reason is that NSAttributedString is now supported, because textColor has no effect if NSAttributedString is set, so I removed textColor.

So my advice is:

Option One, use NSAttributedString:

```swift
let text = NSAttributedString(string: "MESSAGE TEXT", attributes: [
    .foregroundColor: UIColor(red: 239.0/255, green: 169.0/255, blue: 4.0/255, alpha: 0.95)
])

self.showMessage(text, type: .warning, options: [
    .animation(.slide),
    .animationDuration(0.3),
    .autoHide(true),
    .autoHideDelay(3.0),
    .height(60.0),
    .hideOnTap(true),
    .position(.top),
    .textAlignment(.center),
    .textNumberOfLines(1),
    .padding(.init(top: 10, left: 30, bottom: 10, right: 30))
])
```

Option two, use the previous version `1.4.2`, the new version `1.5.0` only changed support for NSAttributedString and removed textColor.

**Comment by wxxsw** [comment]
I may be wrong, I will fix this problem.

**Comment by wxxsw** [comment]
Sorry, I should not delete `textColor` option, and now I've added it back and released a new version `1.5.1`, please update it.

**Comment by jobinsjohn** [comment]
Its fixed....
